free-courses-en.md

```diff
@@ -137,6 +139,11 @@
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046)
 
 
+### jQuery
+
+* [CodeSchool - Try jQuery](https://www.codeschool.com/courses/try-jquery)
```

Comment:
Could you please move this link to `free-programming-interactive-tutorials-en.md`?